src/main/scala/fm/http/server/HttpServerApp.scala

```diff
@@ -325,13 +325,17 @@ abstract class HttpServerApp extends Logging {
    * Note: USR1 is already being used by the JVM for something:
    *  "java.lang.IllegalArgumentException: Signal already used by VM or OS: SIGUSR1"
    */
-  private def registerPingSignalHandlers(server: HttpServer): Unit = {
+  private def registerPingSignalHandlers(server: HttpServer): Unit = try {
     Signal.handle(new Signal("USR2"), new SignalHandler{
       def handle(signal: Signal): Unit = {
         logger.info("Caught USR2 Signal - Disabling /_ping")
         server.disablePing()
       }
     })
+  } catch {
+    case ex: Throwable =>
+      // Cannot register signal handlers (on Windows, or sun.misc.Signal unavailable)
+      val isWin = System.getProperty("os.name", "") startsWith "Windows"
+      if (!isWin) throw ex
```

Comment:
We should either not attempt to call `registerPingSignalHandlers` or have the first line of the method return if it's running under windows.  I'm not sure why we would do a try/catch here

Reply:
I was mostly just hoping to avoid the check, since it's the non-standard use pattern but since this only gets called on startup I'm fine with just doing the check for every OS all the time